server/app/views/v1/domain_authorizations/_domain_authorization.json.jbuilder

```diff
@@ -4,6 +4,7 @@ json.status authorization.status
 json.challenge authorization.challenge
 json.challenge_opts authorization.challenge_opts
 json.authorization_type authorization.authorization_type
+json.expires authorization.expires
```

Comment:
Should it be `expires_at` ?

Reply:
Yes, it should... and I figure the model field should also be `expires_at`?

Reply:
Changed both the JSON API and the model to use `expires_at`.